packages/create-cloudflare/templates-experimental/next/templates/wrangler.toml

```diff
@@ -0,0 +1,12 @@
+#:schema node_modules/wrangler/config-schema.json
+name = "<app-name>"
+main = ".worker-next/index.mjs"
+
+compatibility_date = "2024-08-29"
+compatibility_flags = ["nodejs_compat_v2"]
```

Comment:
with https://github.com/cloudflare/workers-sdk/issues/6821 fixed, could we now drop the `_v2` suffix?